src/asynk/queries/find_task_by_uniq_hash.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM fang_tasks WHERE uniq_hash = $1 LIMIT 1
```

Comment:
I think we should also add `state = new or state = in_progress` filter

Reply:
[comments suggestions](https://github.com/ayrat555/fang/pull/71/commits/4eb86a2e2a6a2e33153736332de9191e4a162d56)